source/Octopus.Server.Client/AppDomainScanner.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+
+namespace Octopus.Client
+{
+    internal static class AppDomainScanner
```

Comment:
Primitive scanner to allow the client's default HTTP route locator service implementation to find all the types implementing HTTP routes for command/request payloads.